### PR TITLE
[PFS] Lib to get feature switches

### PR DESCRIPTION
**Pull request by roopakv**
Add lib to get feature switches.

Fixes: Part of https://app.asana.com/0/1141250355664307/1145091162039545

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [x] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [x] Successfully tested locally (or docker image)
- [ ] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] New and existing unit tests pass locally with my changes
- [x] I have created or modified unit tests to sufficiently cover my changes.

### CODE REVIEW

File: Helper/FeatureSwitch/Definitions.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ */
+class Definitions {
+    const NAME_KEY="name";
+    const VAL_KEY="value";
+    const DEFAULT_VAL_KEY="default_value";
+    const ROLLOUT_KEY="rollout_percentage";
+
+    const M2_SAMPLE_SWITCH_NAME = "M2_SAMPLE_SWITCH";
+
+    const SWITCHES = array(
```
Comment (daisy1754) [inline Helper/FeatureSwitch/Definitions.php]: how about DEFAULT_SWITCH_VALUES?

File: Helper/FeatureSwitch/Definitions.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ * Class Definitions
+ * @package Bolt\Boltpay\Helper\FeatureSwitch
+ */
+class Definitions {
+    const NAME_KEY="name";
+    const VAL_KEY="value";
+    const DEFAULT_VAL_KEY="default_value";
+    const ROLLOUT_KEY="rollout_percentage";
+
+    const M2_SAMPLE_SWITCH_NAME = "M2_SAMPLE_SWITCH";
```
Comment (daisy1754) [inline Helper/FeatureSwitch/Definitions.php]: can you add comment what is this (is it example switch for testing)?

File: Helper/FeatureSwitch/Definitions.php (hunk 1 above)
Comment (roopakv) [reply Helper/FeatureSwitch/Definitions.php]: makes sense.

File: Helper/FeatureSwitch/Definitions.php (hunk 2 above)
Comment (roopakv) [reply Helper/FeatureSwitch/Definitions.php]: yes this is until someone adds some switch we will actually use. it used to test this feature overall :)

File: Helper/FeatureSwitch/Definitions.php (hunk 1 above)
Comment (roopakv) [reply Helper/FeatureSwitch/Definitions.php]: done

**Pull request merged**